packaging/rpm/microshift.spec

```diff
@@ -109,7 +109,7 @@ The microshift-selinux package provides the SELinux policy modules required by M
 %package networking
 Summary: Networking components for MicroShift
 Requires: microshift = %{version}
-Requires: openvswitch3.1
+Requires: (openvswitch3.1 or openvswitch)
```

Comment:
I'm a bit uneasy with this change because we're basically allowing unversioned package installation for `openvswitch`. We do not support Fedora, so I would vote not to make this change.

Reply:
Hi @ggiguash , I could even change this to 
```
(openvswitch3.1 or openvswitch >= 3.1)
```

but that's still nice to have if that can just help to let the installation go, although in a non-officially-supported scenario.

Reply:
That code would certainly be better. 
@dhellmann / @pacevedom , do you have any objections from merging this code?
```
(openvswitch3.1 or openvswitch >= 3.1)
```

Reply:
Thanks @ggiguash, let me push it first and run some testing even on Fedora.

Reply:
confirmed as working good in fc38 too.